CityOfDetroit/bloom
# fix: add descriptive alt tags to listings images

# Pull Request Template

## Issue Overview

This PR addresses #944 


## Description

Changed listings images alt text to be more specific (to include listing name)
For `listingCard` (in /listings) it will read listing name and `- photo`, as it is not treated as Image, but link.
For images for specific listing it is treated as an Image, so there will be added `- photo (number of photo)` only when having multiple images (both in listing view, and after modal is opened).

## How Can This Be Tested/Reviewed?

while having screen reader on, go to `/listings` focus on images.
do the same for `listing/:id` view, both for single image, and multiple image listing.
For multiple images test images inside modal (that appears after clicking).

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
